Userland/Libraries/LibJS/Token.h

```diff
@@ -181,7 +182,7 @@ class Token {
 public:
     Token() = default;
 
-    Token(TokenType type, DeprecatedString message, StringView trivia, StringView value, StringView filename, size_t line_number, size_t line_column, size_t offset)
```

Comment:
Why change this one if the underlying `m_message` is still a DeprecatedString?

Feel free to update that as well, combined with some of the changes in Lexer in its own commit.

Reply:
I decided to update Token to use String and it is now here under a separate commit.